TicTacToe.cpp

```diff
@@ -128,22 +128,40 @@ int TicTacToe::checkVictoryVertical(int teamSide) // check all (9) vertical colu
 
 void TicTacToe::viewAll()
 {
-	// Prints out the field with relevant team integers
-	std::cout << "    " << field[2][0][2] << " --- " << field[2][1][2] << " --- " << field[2][2][2] << std::endl;
+	int hold[27];   //create array to hold field integers
+	int count = 0;  //create counter to store integers in the array
+	for (int i = 0; i < 3; i++) {
+	//loop through the field and store integers
+		for (int j = 0; j < 3; j++) {
+			int l = field[i][j].size()
+			for (int k = 0; k < 3; k++) {
+				if(k<l){
```

Comment:
This ignores cases when k == 1 and k == 2

Reply:
I'm not sure what you mean by that - doesn't the else condition cover those cases?